mangaki/mangaki/migrations/0087_auto_20171023_2100.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2017-10-23 21:00
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('mangaki', '0086_add_anilist_creator_id'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserBackgroundTask',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_on', models.DateTimeField(auto_now_add=True)),
+                ('task_id', models.CharField(max_length=80)),
+                ('tag', models.CharField(max_length=80)),
+                ('owner', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='background_tasks', to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.AlterField(
```

Comment:
:'(

Reply:
Why such a sadness?

Reply:
This migration has nothing to do with this PR. It still has to be included because no one did it, it seems like :/.